### ISSUE TITLE
Skip Graphics Card Window

### ISSUE THREAD

**Issue by ghost** [cursey/kanan#35]
There used to be a mod to skip the following window that appears if the user's gfx card does not meet the recommended specs:

![image](https://cloud.githubusercontent.com/assets/11168098/17423320/bfd1999c-5a6e-11e6-9f5b-50a1b04ff176.png)

That mod no longer works, unfortunately.
Do you think you could make a new one to do so? :+1: 

Thanks :)

**Comment by cursey** [comment]
How early does this message appear after starting mabi?

**Comment by ghost** [comment]
Usually within a few seconds. After launching, but before the blue NGS pop-up.

**Comment by cursey** [comment]
That patch would work if kanan attaches in time.  Give it a try.

**Comment by x99user** [comment]
yeah, i'm not entirely sure if it belongs in delayed or not.

**Comment by ghost** [comment]
Actually, I forgot about something: Kanan doesn't attach to Mabi until _after_ the graphics message...
So while the patch works (tested with mod_sharker), it doesn't work with Kanan. :(

**Comment by cursey** [comment]
Yeah, I figured that would be the case.  I had to implement multi-client and data folder in such a way that timing didn't matter.  I might think of something though.
